eos/constraints/dispersive-bounds.yaml

```diff
@@ -0,0 +1,97 @@
+###########################
+# b -> c unitarity bounds #
+###########################
+# {{{
+b->c::Bound[0^+]:
+    type: UniformBound
+    observables: [ 'b->c::Bound[0^+]@CLN' ]
+    kinematics: [ {} ]
+    options: [ {} ]
+    bound: 1.0
+    # cf. [BG:2016A], eq. (2.8), p.5
+    uncertainty: 0.0130561
+b->c::Bound[0^-]:
+    type: UniformBound
+    observables: [ 'b->c::Bound[0^-]@CLN' ]
+    kinematics: [ {} ]
+    options: [ {} ]
+    bound: 1.0
+    # using the same relative uncertainty as for 0^+, cf. [BG:2016A], eq. (2.8), p.5
+    uncertainty: 0.0130561
+b->c::Bound[1^+]:
+    type: UniformBound
+    observables: [ 'b->c::Bound[1^+]@CLN' ]
+    kinematics: [ {} ]
+    options: [ {} ]
+    bound: 1.0
+    # cf. [BG:2016A], eq. (2.8), p.5
+    uncertainty: 0.0093549
+b->c::Bound[1^-]:
+    type: UniformBound
+    observables: [ 'b->c::Bound[1^-]@CLN' ]
+    kinematics: [ {} ]
+    options: [ {} ]
+    bound: 1.0
+    # same relative uncertainty as for 1^+, cf. [BG:2016A], eq. (2.8), p.5
+    uncertainty: 0.0093549
+# }}}
+
+###########################
+# b -> s unitarity bounds #
+###########################
+# {{{
+Lambda_b->Lambda(1520)::Bound[0^+_V]:
```

Comment:
This works, since you use a specific name for this constraint. A global bound, using all available transitions, will be much tougher to organise consistently. I do wonder if we should enforce the unitarity bounds to always be part of ``manual_constraints``.

Reply:
Why is a problem? Not sure I understand the problem with e.g.
```yaml
b->s::Bound[0^+_V]:
    type: UniformBound
    observables:
        - 'B_s0::Saturation[0^+_V]'
        - 'Lambda_b->Lambda(1520)::Saturation[0^+_V]@ABR2022'
        - 'Lambda_b->Lambda::Saturation[0^+_V]@BMRvD2022'
        - 'B->K::Saturation[0^+_V]@BSZ2015'
        - ...
    kinematics: [ {}, {} ]
    options: [ {}, {} ]
    bound: 1.0
    uncertainty: 0.1
```

Reply:
This is not a technical problem, to be discussed off line.